### DISCUSSION TITLE
LogStash 5044 port cant shown

### DISCUSSION THREAD

**Discussion by chowdary709** [learndevopsonline/learndevopsonline#1871] [Q&A]
![Screenshot (8)](https://github.com/learndevopsonline/learndevopsonline/assets/149947513/4f4c716c-13d4-425d-b2cd-db3eec5d5eb0)

**Comment by chowdary709** [comment]
@chaithanya1812  im install logstash (elk) in labauto  
 Logstash port (5044) number cannot showing  you can try it ones more

**Comment by chaithanya1812** [comment]
Hi @chowdary709, can you share new public-ip of your instance...
